packages/base/build.gradle.kts

```diff
@@ -32,6 +32,16 @@ elide {
   }
 }
 
+kotlin {
+  sourceSets {
+    val nativeMain by getting {
+      dependencies {
+        implementation("org.jetbrains.kotlinx:atomicfu:0.23.1")
```

Comment:
nit: add a comment explaining this pin, and use a version catalog instead if possible